python/ray/experimental/util/types.py

```diff
@@ -9,6 +9,23 @@ class _CollectiveOp(Enum):
 
 @PublicAPI
 class ReduceOp(_CollectiveOp):
+    pass
+
+
+@PublicAPI
+class AllReduceReduceOp(ReduceOp):
+    SUM = 0
```

Comment:
Are you saying you have to define {SUM, PRODUCT, ...} manually in both all reduce and reduce scatter ops?